exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,13 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (number < 0) {
+    return "negative";
+  } else if (number >= 0) {
+    return "positive";
+  }
+}
```

Comment:
1. If there is only 2 outcomes (e.g. negative or positive; true or false). We usually use if / else. **else if** is used when there is 3 or more outcomes.
2. You can use the **? :** expression. Refer E-conditional exercise above.